react/components/ProductName/index.js

```diff
@@ -101,7 +101,7 @@ class ProductName extends Component {
     }
 
     return (
-      <Wrapper className={productName.productNameContainer}>
+      <Wrapper className={classNames('mv0', productName.productNameContainer, className)}>
```

Comment:
change the order
`(productName.productNameContainer, 'mv0', className)`

Reply:
Fixed.